integration-tests/test-cases/scenario-tests/src/test/java/io/cellery/integration/scenario/tests/petstore/PetStoreTestCase.java

```diff
@@ -47,26 +47,17 @@
     private static final String VERSION = "latest";
     private static final String LINK = "petStoreBackend:pet-be-inst";
     private WebDriver webDriver;
-    private WebDriverWait wait;
     private User alice;
     private User bob;
     private Cart cart;
     private PetAccessory[] itemsOfAlice;
     private Order order;
-    private String idpLogoutHeader = "OPENID CONNECT LOGOUT";
-    private String petStoreSignInWebContent = "SIGN IN";
-    private String identityServerHeader = "OPENID USER CLAIMS";
-    private String personalInformationHeader = "Pet Store";
-    private String noOrdersPlaced = "No Orders Placed";
-    private String petCarrierCageXpath = "//*[@id=\"app\"]/div/main/div/div[2]/div/div[1]/div/div[4]/div[2]/button";
-    private String boneShapedToyXpath = "//*[@id=\"app\"]/div/main/div/div[2]/div/div[4]/" +
-            "div/div[4]/div[2]/button";
 
     @BeforeClass
     public void setup() {
         WebDriverManager.getInstance(CHROME).setup();
         webDriver = new ChromeDriver(new ChromeOptions().setHeadless(true));
-        wait = new WebDriverWait(webDriver, 120);
+        WebDriverWait wait = new WebDriverWait(webDriver, 120);
```

Comment:
Let's rename the variable name from 'wait' to 'webDriverWait'

Reply:
Updated.